src/script.js

```diff
@@ -5,79 +5,29 @@
 
 //HTML Elements
   const container1 = document.getElementById("result1Box");
-  console.log("Container 1 is: ", container1)
   const container2 = document.getElementById("result2Box");
-  console.log("Container 2 is: ", container2)
   const form = document.querySelector("form");
-  console.log("Form is: ", form)
 
-//Functions
+//Event Listeners
   //Get User Input
+    form.addEventListener("submit", async (event) => {
+      event.preventDefault();
+      const formData = new FormData(form);
+      await pullFromAPI(shakespeare, formData, container1);
```

Comment:
To Do:

- Create an eventListener to determine which API was chosen
- Embed a second call of this function with container2 as the 3rd argument